source/collimation.f90

```diff
@@ -1400,6 +1645,11 @@ subroutine collimate_parseInputLine(inLine, iLine, iErr)
     call cdb_addFamily("tdi",    nSigIn(13),famID,fErr)
 
   case(5)
+    if(nSplit /= 10) then
```

Comment:
uh - here the mandatory ones are the first 8 (i.e. TCTs), whereas (almost) nobody cares about XRPs (also in declaring them in CollDB and fort.3)

Reply:
Sure. I just required all values to be used by default. We can modify the requirement now that all values are actually given a default, which wasn't the case before.